.github/workflows/test.yml

```diff
@@ -22,5 +25,8 @@ jobs:
       - name: Install npm dependencies
         run: npm install
 
+      - name: Run linter
+        run: npm run lint
```

Comment:
Should we have another workflow for lints stuff?

Reply:
Don't think we should, if the lint fails, the test would fail as well. And that seems reasonable right?

Reply:
makes sense